src/sass/table/table.scss

```diff
@@ -64,6 +68,34 @@ $density-relaxed-img-radius: 8px;
     padding: $density-relaxed-padding;
 }
 
+.table--frozen-header thead {
+    position: sticky;
```

Comment:
Does this solution also work with dropdowns inside the cell?

Reply:
Good catch. Let me implement the full markup inside with open dropdowns and check. If it doesn't, I'll check for a `css` solution, but we may end up having to do something via `js` in ebay ui.

Reply:
@agliga , it works fine, at least with listbox. I have a feeling the issues the other team was having had to do with incorrect `z-index` and lack of `background-color` on `<thead>` and all other non-data columns (outer wrapper).

<kbd><img width="956" alt="image" src="https://github.com/user-attachments/assets/db879389-62ed-4096-adfd-697c26963c30"></kbd>